### DISCUSSION TITLE
how to customize service name without agent

### DISCUSSION THREAD

**Discussion by houbinglei** [apache/skywalking#10468] [Q&A]
excuseme~ 
i have seen this doc:
If the service name includes double colons (::), the literal string before the colons is taken as the group name. In the latest GraphQL query, the group name has been provided as an optional parameter.

 Now i use istio 1.16x with the conf:
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  meshConfig:
    defaultProviders:
      tracing:
      - "skywalking"
    enableTracing: true
    extensionProviders:
    - name: "skywalking"
      skywalking:
        service: tracing.istio-system.svc.cluster.local
        port: 11800
i need to set the service name with xxx::xxx ; but the service name is auto genarated ; and  there is no SkyWalking agent installed to the services, all the traces are generated by the sidecar proxies; could i customize the service name ? 
![企业微信截图_b6945850-cf8f-4cbf-8370-456ffc72b832](https://user-images.githubusercontent.com/37292926/222031250-78d661c7-4fde-4fc5-a00a-2c7e9e96179b.png)

**Comment by wu-sheng** [comment]
Duplicated https://github.com/apache/skywalking/discussions/10435#discussioncomment-5161862. Don't do this.
Locked.
